implementations/rust/ockam/ockam_command/src/reset/static/after_long_help.txt

```diff
@@ -0,0 +1,6 @@
+# Removes the local Ockam configuration including all Spaces, Identities, and Nodes
```

Comment:
One note here. By default, the command will only remove the local state, basically the `database.sqlite3` file at `$OCKAM_HOME`. 

If you pass the `--all` argument, it will also try deleting the orchestrator resources (i.e. spaces and projects).